scripts/install_theme_vendor.sh

```diff
@@ -0,0 +1,59 @@
+#!/usr/bin/env bash -e
+
+VENDOR_DIR="mkdocs-material/material/assets/vendor"
+rm -rf "$VENDOR_DIR"
+
+# MathJax
+TEMP_DIR="$(mktemp -d -t "download-mathjax-XXXXXXXX")"
```

Comment:
And I think you can set a different name for TEMP_DIR, and you can create a function that will delete all temporary directories and then set it as a TRAP function. This ensures that the temporary directory is deleted when the script exits normally and unexpectedly. e.g.
```bash
function _when_exit() {
	rm -rf "$MATHJAX_TEMP_DIR"
	rm -rf "$ICONS_TEMP_DIR"
}
trap _when_exit EXIT
```

Reply:
It's by design. If the script exits with error we can check the `$TEMP_DIR` to see what's wrong.

Reply:
> It's by design. If the script exits with error we can check the `$TEMP_DIR` to see what's wrong.

Okay, you can TRAP another function to tell user to check it. It should be bind to `ERR` (means it will execute when script exits unexpectedly).
```bash
trap "{ echo 'Oops! You should ....'; }" ERR
```